bots/example.py

```diff
@@ -0,0 +1,47 @@
+import os
+from datetime import timedelta
+
+from ape_ethereum import multicall
+from silverback import SilverbackApp
+
+from apepay import StreamManager
+
+app = SilverbackApp()
+# NOTE: You should use one bot per-supported network
+# NOTE: This bot assumes you use a new bot per deployment
```

Comment:
suggestion: What is this noting? Why does this matter? Might be worth expanding on this if it's important for a user to be aware of.